Server/config/mongo.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require("mongoose");
+require("dotenv/config");
+
+const connectoToMongo = function(){
+    
+    mongoose.connect(process.env.MONGO_HASH,
+
+        { useUnifiedTopology: true,useNewUrlParser: true }).then(()=>{
+        
+        console.log(`Succesfully connected`);
+        
+        }).catch(error=>{
+        
+        console.log(String(error.message));
+        
```

Comment:
return something like server error

Reply:
Unnecessary console.logs removed and returned something to the server